TwitterHashtagScrape.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+import tweepy
+import json
+from tweepy import api
+
+access_token="" #Add your token
+access_token_secret="" #Add your access_token_secret token
+
+api_key="" #Add your api
+api_key_secret="" #Add your api_key_secret token
+
+auth = tweepy.OAuthHandler(consumer_key=api_key,consumer_secret=api_key_secret)
+auth.set_access_token(access_token,access_token_secret)
+
+api = tweepy.API(auth)
+print
+
+#def write_Json(data, filename="hashtag.json"):
```

Comment:
What's the problem with your JSON function code why you commented on it?

Reply:
Okay if that's the issue with the above code format I can re-write the changes